### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by v-zhuravlev**

### CODE REVIEW

File: CODEOWNERS
Comment (Dasomeone) [inline CODEOWNERS:18]: Missing newline

Comment (v-zhuravlev) [reply CODEOWNERS:18]: it was added actually

File: CODEOWNERS
Comment (v-zhuravlev) [reply CODEOWNERS:7]: oops

**Review by Dasomeone — APPROVED** [review]
Hmm not sure why it's showing the newline issue in CODEOWNERS, but let's not block on it

**Pull request merged**
